server/sonar-web/src/main/js/apps/projectActivity/components/GraphHistory.js

```diff
@@ -44,23 +45,30 @@ type Props = {
   updateSelectedDate: (selectedDate: ?Date) => void,
   updateTooltip: (selectedDate: ?Date) => void
 };
+*/
 
+/*::
 type State = {
   tooltipIdx: ?number,
   tooltipXPos: ?number
 };
+*/
 
 export default class GraphHistory extends React.PureComponent {
-  props: Props;
-  state: State = {
+  /*:: props: Props; */
+  state /*: State */ = {
     tooltipIdx: null,
     tooltipXPos: null
   };
 
-  formatValue = (tick: string | number) =>
+  formatValue = (tick /*: string | number */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)